scripts/flash.sh

```diff
@@ -312,19 +310,18 @@ arduino-cli config init
 # update the core index (see arduino-cli documentation)
 arduino-cli core update-index
 
-# get the device port, board, fqbn, and core
-get_board_info
-
-# install this device's core
-arduino-cli core install $DEVICE_CORE
-
 # symlink libraries
-get_lib_install_dir
```

Comment:
Why did we change the flash script? Don't you need to determine what directory to install the libraries before symlinking them? They are different on different Linux distros and on Macs, so this function is important!

Reply:
I guess its a bit weird, but this is already called in the `clean` function, and we always `clean` before doing anything else so no need to call it twice.

Reply:
Ah okay, yea then this is I guess a question of whether we should `clean` before doing anything each time, but for now this makes sense to delete this.